docs/DeveloperGuide.md

```diff
@@ -711,16 +711,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes from step 2.
 
-**Use case UCV04: Clear all volunteers in volunteer list**
+**Use case UCEV01: List all events joined by a volunteer**
```

Comment:
should be `UCEV04` while max's suggestions are `UCEV01` - `UCEV03`

Reply:
wait when did i suggested hahahha

Reply:
as in for ur ones change to UCEV01-UCEV03 (refer to my PR)

Reply:
oh okeh

Reply:
oki